### ISSUE TITLE
Yet Another Issue with spongeforge and AE2

### ISSUE THREAD

**Issue by NocturnalSergal** [SpongePowered/SpongeForge#1470]
**I am currently running...**

- SpongeForge version: Latest as of April 27th, 2017
- Forge version: 2254
- Java version: 8u121
- Operating System:

 - [x] Linux (Ubuntu, Debian, Fedora, etc)
 - [ ] macOS
 - [ ] Windows

- Plugins/Mods: 

Minecraft, Minecraft Coder Pack, Forge Mod Loader, Minecraft Forge, Applied Energistics 2 Core, Aroma1997Core, Not Enough Items, OpenComputers (Core), SpongeAPI, SpongeForge, CoFH ASM, FoamFixCore, ccl-entityhook, CoFH Core, Actually Additions, IndustrialCraft 2, Advanced Machines, Advanced Solar Panels, Applied Energistics 2, BD Lib, AE2 Stuff, InfinityLib, AgriCraft, Baubles, Botania, Akashic Tome, Just Enough Items, AppleSkin, Aquaculture, ArchitectureCraft, Aroma1997Core|Helper, Aroma1997's Dimensional World, Morph-o-Tool, Psi, Quark, AutoRegLib, Bed Bugs, Better Builder's Wands, BiblioCraft, Guide-API, Blood Magic: Alchemical Wizardry, Blood Arsenal, Bookshelf, CodeChicken Lib, Brandon's Core, Ceramics, Chameleon, Chest Transporter, Chisel, MCMultiPart, Chisels & Bits, ShadowMC, Clipboard, CodeChicken Core, Compact Solar Arrays, Cooking for Blockheads, CraftTweaker JEI Support, MineTweaker 3, Dark Utilities, PTRModelLib, Decocraft, Forestry, ElecCore, EnderStorage, OpenComputers, DeepResonance, Draconic Evolution, Storage Drawers, Drawers & Bits, Elevator Mod, Embers, EnderCore, Ender IO, Engineer's Workshop, Valkyrie Lib, Environmental Tech, ET Lunar, Extra Utilities 2, Zero CORE, Extreme Reactors, Flat Colored Blocks, SonarCore, FluxNetworks, FoamFix, Forgelin, Forge Multipart CBE, Forge Microblocks, Minecraft Multipart Plugin, FTBLib, FTBUtilities, Funky Locomotion, Funnels, GenDustry, Gravestone, Gravitation Suite, Hammer Core, HelpFixer, Hopper Ducts, Mantle, Tinkers' Construct, The One Probe, Immersive Engineering, LunatriusCore, InGame Info XML, Interaction Wheel, Inventory Tweaks, Iron Backpacks, Iron Chest, JEI Bees, JourneyMap, KleeSlabs, Long Fall Boots, MalisisCore, MalisisDoors, Mercurius, Modular ForceField System, MmmMmmMmmMmm, mob_grinding_utils, MTLib, Mod Tweaker, Railcraft, More Bees, Morpheus, ModPack Utilities, ModPack Basic Tools, MrTJPCore, ShetiPhian-Core, Multi-Storage, Natura, Nether Portal Fix, Open Glider, p455w0rd's Library, Pam's HarvestCraft, ProjectRed Core, ProjectRed integration, ProjectRed Transmission, ProjectRed Illumination, ProjectRed Expansion, projectred-tansportation, ProjectRed Exploration, RebornCore, QuantumStorage, Ranged Pumps, reborncore-MCMultiPart, Thermal Foundation, Redstone Arsenal, Refined Relocation, Refined Storage, Reliquary, RFTools, RFTools Control, RFTools Dimensions, Roots, Signals, SimpleVoidWorld, Simple Retrogen, Simply Jetpacks 2, Sleeping Bag, Snad, Solar Flux Reborn, Steve's Carts 2, Storage Drawers Extras, Super Circuit Maker, Super Crafting Frame, TechReborn, Thermal Expansion, Tinkers Tool Leveling, TOP Addons, TorchMaster, Translocator, Useful Nullifiers, WanionLib, Wireless Crafting Grid, Wireless Crafting Terminal, Woot, WirelessRedstone-CBE, XNet, YABBA, UniDict

**What is the expected result?**

For the event to be tracked

**What is the current result?**

said event isnt being tracked and sponge is throwing an error the size of kansas

https://pastebin.com/Vrt7YM3b

(this is the error, if the whole log is needed ill need to cull about 1 million repetions of this error from it)

**Comment by gabizou** [comment]
If you can provide the startup of the server log, that'd be wonderful, and actually list out what `/sponge version` is for SpongeForge as "latest" is not helpful.

**Comment by NocturnalSergal** [comment]
I completely forgot that command existed 

SpongeForge-1.10.2-2281-5.2.0-BETA-2274 
SpongeAPI: 5.2.0-SNAPSHOT-a3257a0

And the log file is in its entirety since I just found some more debug stuff and I can't really edit it on my phone that well. Probably should have uploaded it yesterday 

https://www.dropbox.com/s/8z7gtct187tibp0/fml-server-3.log?dl=0

**Comment by Grauldon** [comment]
Hello @NocturnalSergal 

I'm not real experienced at this, but I'm getting back into coding/debugging and I hope this helps.

My first observation is you are running Forge Mod Loader version 12.18.3.**2254** with spongeforge-1.10.2-**2281**-5.2.0-BETA-2274.jar.  The version of FML does not match SpongeForge.  Getting the 12.18.3.2281 of FML may solve the problem.

In addition, I noticed while verifying file names that spongeforge-1.10.2-2281-5.2.0-BETA-2309 came out yesterday.  I'm not sure, but the title suggests this file may address your issue.

Last, I used grep on your log file and [uploaded lines][a] that you and @gabizou may be interested in.  They may or may not be part of the problem, but I thought I would point them out.  Based on [SpongeDocs' Debugging page](<https://docs.spongepowered.org/stable/en/server/spongineer/debugging.html>), I think the first two lines are related to the version mismatch.

[a]: <https://pastebin.com/hpU5XKms>
